server/bin/pbench-unpack-tarballs.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+import os
+from pathlib import Path
+import sys
+
+from pbench.common.exceptions import BadConfig
+from pbench.common.logger import get_pbench_logger
+from pbench.server import PbenchServerConfig
+from pbench.server.database import init_db
+from pbench.server.unpack_tarballs import UnpackTarballs
+
+_NAME_ = "pbench-unpack-tarballs"
+
+# The link source for the operation of this script.
+_linksrc = "TO-UNPACK"
+
+
+def main(cfg_name):
+    if not cfg_name:
+        print(
+            f"{_NAME_}: ERROR: No config file specified; set"
+            " _PBENCH_SERVER_CONFIG env variable or use --config <file> on the"
+            " command line",
+            file=sys.stderr,
+        )
+        return 2
+
+    try:
+        config = PbenchServerConfig(cfg_name)
+    except BadConfig as e:
+        print(f"{_NAME_}: {e}", file=sys.stderr)
+        return 1
+
+    logger = get_pbench_logger(_NAME_, config)
+
+    # We're going to need the Postgres DB to track dataset state, so setup
+    # DB access.
+    init_db(config, logger)
+
+    BUCKET = str(sys.argv[1])
```

Comment:
A technicality, but `bucket` should be lowercase; it's not a "constant" any more than anything else here.

Reply:
I'd still prefer lowercase, as it's set dynamically by the code from external input rather than statically defined, and I think that warping of standard convention is unnecessarily confusing. But since nobody else seems to be interested and it's merely one of many harbingers of the end of the world as we know it, I'm resolving this conversation. 😆